### DISCUSSION TITLE
Use case video duration

### DISCUSSION THREAD

**Discussion by Deenuji** [microsoft/hack-together-dotnet#42] [Q&A] [answered]
@aaronpowell @daisychaussee  - We are in the process of creating a demo for submission to a hackathon, and it currently runs for approximately 7 minutes. This duration includes the integration of over 4 AI Builder models and more than 8 different OpenAI models  for our developed solution. Is the current video duration within the acceptable range?

**Comment by codemillmatt** [comment, marked as answer]
Hi @Deenuji - that's perfectly fine. Looking forward to seeing what you came up with!
